### ISSUE TITLE
Turn off autoscroll to latest reaction in a topic (motie)

### ISSUE THREAD

**Issue by Socioneers** [kgish/discourse-pvda-plugin#52]
The discourse system automatically scrolls down to latest reaction in a topic (motie). Can we turn this behaviour of with moties?

**Comment by kgish** [comment]
Yes, I will disable this. Is there already a system setting available via the admin dashboard? I will check that first.

**Comment by kgish** [comment]
Turns out that discourse remembers the last position of each user and when he/she returns to the same page will be automatically relocated to that position.

See: https://meta.discourse.org/t/how-should-i-disable-automatic-redirection-to-the-last-reply/21402

**Comment by Socioneers** [comment]
Maybe better to park this issue

**Comment by kgish** [comment]
Okay, moved to parked.
